doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -346,6 +352,12 @@ PMIC samples
 
 |no_changes_yet_note|
 
+nRF RPC
+-------
+
+* Added:
+  * Protocol serialization client sample.
```

Comment:
```suggestion
* Added the protocol serialization client sample.
```
as above - let's align this section alphabetically

Reply:
fixed.